config/serverConfig.js

```
@@ -0,0 +1,95 @@
+      return req.path === '/health';
+    }
+  };
+
+  // Configurazione sicura per rate limiting con trust proxy
+  if (trustProxy) {
+    // Quando trust proxy è abilitato, usa una configurazione più sicura
+    config.keyGenerator = (req) => {
+      // Usa l'IP più specifico disponibile, fallback a req.ip
+      const forwarded = req.get('X-Forwarded-For');
+      const realIp = req.get('X-Real-IP');
+      const clientIp = realIp || (forwarded && forwarded.split(',')[0].trim()) || req.ip;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)